tests/utils/testutils_lc.py

```diff
@@ -163,11 +174,24 @@ def spawn_container(self,
         # TODO: Mount library in docker container
         # self._download_azure_functions()
 
+        # Download python extension base package
+        ext_folder = self._download_extensions()
+
         container_worker_path = (
             f"/azure-functions-host/workers/python/{self._py_version}/"
             "LINUX/X64/azure_functions_worker"
         )
 
+        base_ext_container_path = (
+            f"/azure-functions-host/workers/python/{self._py_version}/"
+            "LINUX/X64/azure/functions/extension/base"
```

Comment:
We'll need to change this to match the new base directory structure (azurefunctions/extension/base as of right now)